src/utils/mtev_log.c

```diff
@@ -2826,6 +2843,13 @@ mtev_log_init_globals(void) {
     mtev_hash_init_locks(&mtev_logops, MTEV_HASH_DEFAULT_SIZE, MTEV_HASH_LOCK_MODE_MUTEX);
 
     filter_runtime = mtev_logic_exec_alloc(&flatbuffer_log_filter_ops);
+
+    mtev_stats_init();
+    mtev_log_lines_stats_ns = mtev_stats_ns(
+      mtev_stats_ns(mtev_stats_ns(NULL, "mtev"), "log"), "lines");
+    stats_ns_add_tag(mtev_log_lines_stats_ns, "framework", "libmtev");
```

Comment:
I would think the mtev:log tag should be added on the "log" namespace,
and the units added on the lines namespace (as it is)

The framework is unecessary b/c it is already on the "mtev" namespace.